app/src/main/res/values/colors.xml

```diff
@@ -28,13 +29,16 @@
         <item>@color/deep_purple_600</item>
         <item>@color/purple_600</item>
         <item>@color/pink_600</item>
+        <item>@color/brown_700</item>
         <item>@color/grey_800</item>
+        <item>@color/grey_600</item>
```

Comment:
Since we are already changing things, I think we should go ahead and add all the colors from the Material Design palette. More specifically, we are still missing Indigo, Blue, Light Blue, Light Green and Amber.